app/src/androidTest/java/com/epfl/beatlink/ui/library/LibraryScreenTest.kt

```diff
@@ -5,27 +5,27 @@ import androidx.compose.ui.test.assertTextEquals
 import androidx.compose.ui.test.junit4.createComposeRule
 import androidx.compose.ui.test.onNodeWithTag
 import androidx.compose.ui.test.performClick
-import androidx.navigation.NavDestination
-import androidx.navigation.NavHostController
+import com.android.sample.ui.library.LibraryScreen
```

Comment:
The import doesn't match the package structure.

```suggestion
import com.epfl.beatlink.ui.library.LibraryScreen
```